arch/arm/boot/dts/overlays/Makefile

```diff
@@ -247,6 +247,8 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	w1-gpio.dtbo \
 	w1-gpio-pullup.dtbo \
 	w5500.dtbo \
+	waveshare-can-fd-hat-mode-a-overlay.dts \
```

Comment:
This should be `waveshare-can-fd-hat-mode-a.dtbo \`, with the same change required for the next line.

Reply:
changed